app/helpers/publishers_helper.rb

```diff
@@ -154,10 +155,14 @@ def publisher_converted_last_settlement_balance(publisher)
     I18n.t("helpers.publisher.conversion_unavailable", code: settlement_currency)
   end
 
-  def publisher_last_settlement_date(publisher)
+  def publisher_last_settlement_date(publisher, locale)
     last_settlement_balance = publisher.wallet&.last_settlement_balance
     if last_settlement_balance&.timestamp.present?
-      Time.at(last_settlement_balance.timestamp).to_datetime.strftime("%B %d, %Y")
+      if locale.to_sym == :ja
+        I18n.l(Time.at(last_settlement_balance.timestamp).to_date, format: :long, locale: :'ja')
```

Comment:
To prevent the locale-specific if statement - can we simplify by passing in the locale into the `I18n.l`?